diff --git a/node_modules/react-native-screens/.DS_Store b/node_modules/react-native-screens/.DS_Store
new file mode 100644
index 0000000..b7fa5d4
Binary files /dev/null and b/node_modules/react-native-screens/.DS_Store differ
diff --git a/node_modules/react-native-screens/android/build.gradle b/node_modules/react-native-screens/android/build.gradle
index 482478b..3eacbac 100644
--- a/node_modules/react-native-screens/android/build.gradle
+++ b/node_modules/react-native-screens/android/build.gradle
@@ -38,6 +38,7 @@ def reactNativeArchitectures() {
 }
 
 android {
+    namespace = "com.swmansion.rnscreens"
     compileSdkVersion safeExtGet('compileSdkVersion', 28)
 
     // Used to override the NDK path/version on internal CI or by allowing
@@ -62,6 +63,9 @@ android {
     lintOptions {
         abortOnError false
     }
+    buildFeatures {
+       buildConfig = true
+    }
     compileOptions {
         sourceCompatibility JavaVersion.VERSION_1_8
         targetCompatibility JavaVersion.VERSION_1_8
