diff --git a/node_modules/react-native-safe-area-context/lib/typescript/src/specs/NativeSafeAreaView.d.ts b/node_modules/react-native-safe-area-context/lib/typescript/src/specs/NativeSafeAreaView.d.ts
index 7dc923a..0b9c119 100644
--- a/node_modules/react-native-safe-area-context/lib/typescript/src/specs/NativeSafeAreaView.d.ts
+++ b/node_modules/react-native-safe-area-context/lib/typescript/src/specs/NativeSafeAreaView.d.ts
@@ -1,7 +1,7 @@
 import type { WithDefault } from 'react-native/Libraries/Types/CodegenTypes';
 import type { ViewProps, HostComponent } from 'react-native';
 export interface NativeProps extends ViewProps {
-    mode?: WithDefault<'padding' | 'margin', 'padding'>;
+    mode?: WithDefault<string, 'padding'>;
     edges?: Readonly<{
         top: string;
         right: string;
diff --git a/node_modules/react-native-safe-area-context/lib/typescript/src/specs/NativeSafeAreaView.d.ts.bak b/node_modules/react-native-safe-area-context/lib/typescript/src/specs/NativeSafeAreaView.d.ts.bak
new file mode 100644
index 0000000..0b9c119
--- /dev/null
+++ b/node_modules/react-native-safe-area-context/lib/typescript/src/specs/NativeSafeAreaView.d.ts.bak
@@ -0,0 +1,14 @@
+import type { WithDefault } from 'react-native/Libraries/Types/CodegenTypes';
+import type { ViewProps, HostComponent } from 'react-native';
+export interface NativeProps extends ViewProps {
+    mode?: WithDefault<string, 'padding'>;
+    edges?: Readonly<{
+        top: string;
+        right: string;
+        bottom: string;
+        left: string;
+    }>;
+}
+declare const _default: HostComponent<NativeProps>;
+export default _default;
+//# sourceMappingURL=NativeSafeAreaView.d.ts.map
\ No newline at end of file
